src/main/java/uk/gov/hmcts/divorce/common/config/advice/GlobalExceptionHandler.java

```diff
@@ -52,4 +53,11 @@
             String.format("%s - %s", exception.getMessage(), exception.contentUTF8())
         );
     }
+
+    @ExceptionHandler(DocumentDownloadException.class)
+    public ResponseEntity<Object> handleBadRequestException() {
+        return new ResponseEntity<>(
+            HttpStatus.BAD_REQUEST
```

Comment:
if the download fails it can be due to 403/401 or 5XX ?

Reply:
I suppose this would be handled by handleFeignException method.

Reply:
Changed it to HttpStatus.INTERNAL_SERVER_ERROR.